app/routeConfig.js

```diff
@@ -2,6 +2,8 @@ import NotFound from '@containers/NotFoundPage/Loadable';
 import HomeContainer from '@containers/HomeContainer/Loadable';
 import routeConstants from '@utils/routeConstants';
 import ItunesContainer from '@containers/ItunesContainer/Loadable';
+// import TrackDetails from '@components/TrackDetails/index';
```

Comment:
Remove this commented out code. Otherwise looks good

Reply:
yes sure!